src/routes/acquirerRoutes.ts

```diff
@@ -34,6 +34,18 @@ const extractFormData = (stepData: Step, body: RequestBody ) => {
     return body[stepData.id]
   }
 
+  if (stepData.id === 'project-aims') {
+    return {
+      aims: body['aims'] || '',
+      explanation: body['explanation'] || ''
+    };
+  }
+
+  const textFields = ['project-aims'];
+  if (textFields.includes(stepData.id)) {
+      return body[stepData.id]
+  }
+
```

Comment:
I think we'll be able to do something like this for the pages with a single text field, like [data required](https://dds-e2e.herokuapp.com/request/030-what-data-no-cat) or [impact](https://dds-e2e.herokuapp.com/request/030-how-impact-cat), but this won't do anything at the moment because `project-aims` will always get handled by the code above.

(For the single text field pages we might be able to add them in to the `radioFields` array - which we could rename to `singleFields` or `simplePages` or something that describes that it's a simple page with 1 input, with an input name that matches the step name)

Reply:
Makes sense, thanks for flagging